docs/sources/articles/basics.md

```diff
@@ -164,13 +162,13 @@ TCP and a Unix socket
 Save your containers state to an image, so the state can be
 re-used.
 
-When you commit your container only the differences between the image
+When you commit your container, only the differences between the image
```

Comment:
The original paragraph has something missing.  This is more succinct and fixes the problem:

--->

When you commit your container, Docker only stores the diff (difference) between the source image and the current state of the container's 		 image. To list images you already have, use the `docker images` command.